scripts/getFactoryCollection.cdc

```diff
@@ -0,0 +1,5 @@
+import CollectionFactory from "../contracts/CollectionFactory.cdc"
+
+pub fun main(user: String, maxItems: Int, shard: String, collection: [String]) : CollectionFactory.CollectionReport? {
```

Comment:
I do not quite understand why this needs to send in aditional collections. Is this the script that is run when you look for items in a shard at first?

The way I see it we need 2 scripts:
 1. initial load of a shard, fetches ids, collections and preresolves maxItems ish items. 
 
 2. load subsequent items, in this script you just send in name and the {string:[üint64]} items you need and resolve just those.